src/views/portfolio/projects/ProjectList.vue

```diff
@@ -27,17 +28,81 @@
   import PolicyViolationProgressBar from "../../components/PolicyViolationProgressBar";
   import xssFilters from "xss-filters";
   import permissionsMixin from "../../../mixins/permissionsMixin";
+  import bootstrapTableMixin from "@/mixins/bootstrapTableMixin";
+
+  let api;
+  let route;
+  let showInactiveProjects;
+  let columns;
+
+  function vueFormatterObject (index, row){
+    return {
+      template: `
+              <bootstrap-table ref="table" :columns="columns" :data="data" :options="options"></bootstrap-table>
+              `,
+      mixins: [permissionsMixin, bootstrapTableMixin],
+      methods: {
+        apiUrl: function (uuid) {
+          let url = `${api.BASE_URL}/${api.URL_PROJECT}/${uuid}/children`;
+          let tag = route.query.tag;
+          if (tag) {
+            url += "/tag/" + encodeURIComponent(tag);
+          }
+          let classifier = route.query.classifier;
+          if (classifier) {
+            url += "/classifier/" + encodeURIComponent(classifier);
+          }
+          if (showInactiveProjects === undefined) {
+            url += "?excludeInactive=true";
+          } else {
+            url += "?excludeInactive=" + !showInactiveProjects;
+          }
+          return url;
+        },
+      },
+      data(){
+        return{
+          columns: columns,
+          data: [],
+          options: {
+            url: this.apiUrl(row.uuid),
+            sidePagination: 'server',
+            queryParamsType: 'pageSize',
+            detailView: true,
+            detailFilter: detailFilter,
+            detailFormatter: (index, row) => {
+              return this.vueFormatter(vueFormatterObject(index, row));
+            },
+            onExpandRow: this.vueFormatterInit
+          }
+        }
+      }
+
+    }
+  }
+
+  function detailFilter(index, row){
+    return (Object.prototype.hasOwnProperty.call(row, 'children') && row.children)
```

Comment:
This is currently causing the expand button and an empty table to be displayed, in cases where no child is active:

![Screenshot from 2022-10-18 14-06-53](https://user-images.githubusercontent.com/5693141/196452639-57706b3a-5427-4b32-bcec-0f73320059b0.png)


This appears to resolve it:

```suggestion
    return (Object.prototype.hasOwnProperty.call(row, 'children') && row.children && row.children.some(child => child.active));
```